keybase/platform_test.go

```diff
@@ -12,15 +12,26 @@ import (
 
 type testConfigPlatform struct {
 	config
+	ProgramPath string
+	Args        []string
 }
 
 func (c testConfigPlatform) promptProgram() (command.Program, error) {
-	return command.Program{
-		Path: filepath.Join(os.Getenv("GOPATH"), "bin", "test"),
-		Args: []string{"echo", `{
+	var programPath = c.ProgramPath
```

Comment:
I think more common is `programPath := c.ProgramPath` and `args := c.Args`
I think you can even combine: `programPath, args := c.ProgramPath, c.Args`